controllers/v1/sessions.js

```diff
@@ -9,76 +9,152 @@
 const sessionsHelper = require("../../services/helper/sessions");
 
 module.exports = class Sessions {
-    
-    form() {
-        return new Promise(async (resolve,reject) => {
-            try {
-                const sessionsForm = await sessionsHelper.form();
-                return resolve(sessionsForm);
-            } catch(error) {
-                return reject(error);
-            }
-        })
-    }
 
-    update(req) {
-        return new Promise(async (resolve,reject) => {
-            try {
-                const sessionUpdated = 
-                await sessionsHelper.update(
-                    req.params._id,
-                    req.body
-                );
+    async update(req) {
+        try {
+            if (req.params.id) {
```

Comment:
@rakeshSgr  Why is the logic in controller??

Reply:
@aman-tunerlabs I thought its better to handle here, so we can keep the logic of update and create logic separately.   
Let me know your thoughts on this.